travis-script.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+if [[ -a .git/shallow ]]; then 
+	git fetch --unshallow
```

Comment:
I'm not sure why we need git history, but perhaps it would be faster to set this from the travis file:
```
git:
  depth: 2147483647
```

Reply:
Yeah, that's probably left over from the old PkgDev templates which did this. Your way is nicer.